src/fidasim.f90

```diff
@@ -1114,6 +1119,8 @@ module libfida
     !+ Variable for storing the calculated passive NPA results
 type(SpectralChords), save      :: spec_chords
     !+ Variable containing the spectral system definition
+type(SpectralChords), save      :: igrid_spec_chords
+    !+ Variable containing the spectral system definition
```

Comment:
I would just add a few new fields to the `SpectralChords` type. You also haven't calculated the line of sight intersections for the cylindrical grid. This should be done in the `read_chords` subroutine. Also what happens if the user doesn't provide a `phi` dependence will the passive/cold calculations just not work?